src/client/sandbox/shadow-ui.js

```diff
@@ -393,8 +386,19 @@ export default class ShadowUI extends SandboxBase {
         return names.join(' ');
     }
 
-    getUIStylesheet () {
-        return nativeMethods.querySelector.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet);
+    getUIStylesheetsCopyFragment () {
+        var stylesheets         = Array.prototype.slice.call(
+            nativeMethods.querySelectorAll.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet));
+        var fragment = document.createDocumentFragment();
+
+        if (stylesheets.length)
+            stylesheets.forEach(stylesheet => fragment.appendChild(stylesheet.cloneNode(true)));
```

Comment:
BTW, we **should** do inline copy of arguments because of https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#32-leaking-arguments